runtime/android/runtime_lib/BUILD.gn

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+import("//xwalk/build/version.gni")
+import("//third_party/icu/config.gni")
+
+android_apk("xwalk_runtime_lib_apk") {
+  apk_name = "XWalkRuntimeLib"
+  android_manifest = "AndroidManifest.xml"
+  java_files = []
+  native_libs = [ "$root_out_dir/libxwalkcore.so" ]
+  version_name = xwalk_version
+  version_code = "$xwalk_version_code"
```

Comment:
It is set on purpose to convert integer to string.
In //xwalk/build/version.gni, Line60,  I tried set "string" as the input_conversion of exec_script, but found it returns "8066002/". so I set "value" as the input_conversion and get integer value 8066002.  So I convert it into string here. 
Maybe we need to modify //xwalk/build/android/generate_version_code.py if want to make it totally clean.